configs/recommended.js

```diff
@@ -100,7 +100,20 @@ module.exports = {
     'array-bracket-spacing': ['error', 'never'],
     'object-shorthand': ['error', 'always'],
     'eqeqeq': ['error', 'always'],
-    'indent': 'off',
+    'space-after-keywords': ['error', 'always'],
+    'space-in-parens': ['error', 'never'],
+    'space-in-brackets': ['error', 'always'],
+    'space-before-blocks': ['error', 'always'],
+    'space-before-keywords': ['error', 'always'],
+    'space-infix-ops': 'error',
```

Comment:
just for consistency
```suggestion
    'space-infix-ops': ['error'],
```